docs/learn/learn-parachains-protocol.md

```diff
@@ -0,0 +1,107 @@
+---
+id: learn-parachains-protocol
+title: Parachains' Protocol Overview
+sidebar_label: Protocol Overview
+description: High level view of actors and protocols involved in running parachains on Polkadot.
+keywords: [parachains, application-specific, sharding, protocol, validator, collator, nominator]
+slug: ../learn-parachains-protocol
+---
+
+:::info
+
+This page is a summary of the [Protocol Overview chapter in **The Polkadot Parachain Host Implementer's Guide**](https://paritytech.github.io/polkadot/book/protocol-overview.html)
+
+:::
+
+The aim of the protocol is to carry a parachain's block from authoring to inclusion through a process that can be carried out repeatedly and in parallel for each parachain connected to the relay chain.
+
+## Main Actors
+
+### Validators
+
+They are responsible for validating proposed parachain's blocks by checking the **Proof-of-Validity** (PoV) of the blocks and ensuring the PoV remains available. They have "skin in the game", meaning they have funds bonded on-chain that can be partially or fully confiscated in case of misbehavior.
+
+### Collators
+
+They create the PoV that validators know how to check. Creating PoV requires familiarity with transaction format and block authoring rules of a specific parachain, as well as having access to its full state.
+
+### Fishermen
+
+They catch misbehaving validators in exchange for a bounty. Collators and validators can behave as fishermen.
+
+## Summary View
+
+The parachains' protocol can be summarized into three main steps:
+
+1. Collators send to validators parachain blocks with their PoV for check.
+2. Validators validate the block using the PoV, signing statements that can have positive or negative.
+3. With enough positive statements the block can be added to the relay chain.
+
+The figure below shows a representation of a parachain with collators and validators. The figure also shows the journey of a block through the three main steps, as well as the sections where the [Inclusion Pipeline](#inclusion-pipeline) and the [Approval Process](#approval-process) take place (see next Sections).
+
+![parachain-protocol-summary](../assets/parachain-protocol-summary.png)
```

Comment:
Is this figure adapted from a different source? Typically, the chronological order of events is shown from left to right.

Reply:
I did the figures based on the description on the parity's doc. I noticed that the parachain is always shown from the right (attached to the relay chain on the right side). The idea was to keep the same location and zoom in to show the different processes (like in GIS/maps representations). That is why the figures here are from right to left. It is not clear the zoom part, I will change them a little to make the story clearer.

Reply:
I see! That is the architecture figure, where the position of the components does not matter. What you are describing through your figures is a pipeline (chain of events), so left to right is more intuitive I think.